modules/core/src/main/scala/lucuma/odb/api/model/TargetModel.scala

```diff
@@ -274,6 +279,13 @@ trait TargetOptics { self: TargetModel.type =>
   val siderealTracking: Optional[TargetModel, SiderealTracking] =
     lucumaTarget.composeOptional(gemTargetSiderealTracking)
 
+  // Add to `core` `SiderealTrackingOptics`
+  private val siderealTrackingCatalogIdLens: Lens[SiderealTracking, Option[CatalogId]] =
+    Lens[SiderealTracking, Option[CatalogId]](_.catalogId)(a => b => b.copy(catalogId = a))
```

Comment:
I'll switch to the one I added in `core` as part of upgrading to the latest core.